NFT Preview Card Component/styles.css

```diff
@@ -0,0 +1,144 @@
+@import url("https://fonts.googleapis.com/css2?family=Outfit:wght@300;400;600&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --soft-blue: hsl(215, 83%, 47%);
+  --cyan: hsl(208, 100%, 50%);
+
+  /* ### Neutral */
+
+  --main-bg: hsl(269, 68%, 23%);
+  --card-bg: hsla(269, 55%, 53%, 0.864);
+  --line: hsl(0, 0%, 100%);
+  --white: hsl(0, 0%, 100%);
+}
+
+img {
+  max-width: 100%;
+}
+
+ul {
+  list-style-type: none;
+}
+
+p {
+  font-weight: 300;
+}
+
+body {
+  font-family: "Outfit", sans-serif;
+  font-size: 18px;
+  background-color: var(--main-bg);
+  color: hsl(0, 0%, 100%, 60%);
+  padding: 1.25rem;
+}
+
```

Comment:
Don't target element directly target it with class and id's

Reply:
Hello Sir Committed the Changes requested